_config.yml

```diff
@@ -1,16 +1,16 @@
 # Configurations
-title:            Jekyll Gitbook
-longtitle:        Jekyll Gitbook
-author:           HE Tao
-email:            sighingnow@gmail.com
+title:            Geostyler Cookbook
+longtitle:        Geostyler Cookbook
+author:           Krishna Lodha
+email:            me@krishnaglodha.com
 description: >
-  Build Jekyll site with the GitBook style.
+  Style cookbook for Geostyler.
 
 version:          1.0
 gitbook_version:  3.2.3
 
-url:              'https://sighingnow.github.io'
-baseurl:          '/jekyll-gitbook'
+url:              'https://github.com/geostyler/geostyler-cookbook'
```

Comment:
Non-blocking: We might have to change this url to a github.io url

Reply:
we'll change it later?

Reply:
Yes